src/Uno.Foundation/Rect.cs

```diff
@@ -244,9 +244,9 @@ public void Intersect(Rect rect)
 		public void Union(Rect rect)
 		{
 			var left = Math.Min(Left, rect.Left);
-			var right = Math.Max(left + Width, rect.Right);
+			var right = Math.Max(Left + Width, rect.Right);
```

Comment:
Big difference! Might be breaking ?

Reply:
It was definitely broken and never used in Uno itself.